### DISCUSSION TITLE
Blacklisting items that can be put in the backpack

### DISCUSSION THREAD

**Discussion by 0Prinsipeeeee** [P3pp3rF1y/SophisticatedBackpacks#931] [Q&A]
Hello! I love really love your mod! It's a huge help for modpack makers :D, just asking if it's possible to blacklist items to be put in the backpack? Because I found out that a combination of Iron's RPG Tweaks(removes hunger bar by default) + Artifacts(speciffically eternal steak) + Sophisticated Backpack (the auto eat upgrade ) = the auto eat tool will just keep eating the eternal steak because it has no cooldown when inside the backpack resulting in unli use.

**Comment by P3pp3rF1y** [comment]
`disallowedItems` in the config
